deployment/molecule/default/tests/test_oracle.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pytest
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('oracle')
+
+
+@pytest.mark.parametrize("name", [
+    ("oracle_rabbit_1"),
+    ("oracle_redis_1"),
+    ("oracle_bridge_request_1"),
+    ("oracle_bridge_collected_1"),
+    ("oracle_bridge_affirmation_1"),
+    ("oracle_bridge_senderhome_1"),
+    ("oracle_bridge_senderforeign_1"),
+])
+def test_docker_containers(host, name):
+    container = host.docker(name)
+    assert container.is_running
+
+
+@pytest.mark.parametrize("service", [
+    ("poabridge"),
+    ("rsyslog")
+])
+def test_services(host, service):
+    assert host.service(service).is_enabled
+    assert host.service(service).is_running
+
+
+def test_remote_logging(host):
+    assert host.file('/etc/rsyslog.d/35-remote-logging.conf').exists
```

Comment:
We also need to check that logs are being written in proper place.

Reply:
Unfortunately it cannot be tested - the reason is that we use `docker.sock` mounting:
```
volumes:
      - /var/run/docker.sock:/var/run/docker.sock
```

which is solving the docker-in-docker slowness, but also result in that the actual log files are on host machine running the molecule tests, instead of on `oracle-host`

Reply:
But will it be possible to test the logging if we specify another endpoint for the remote logs? Probably it will complicate the test logic but I am curious if we still have any way to test as much of the deployment functionality as we provide.

Reply:
I'm not sure. I created #120 to investigate and try to achieve this.

Reply:
thanks